ext/MeasurementsRecipesBaseExt.jl

```diff
@@ -19,8 +19,21 @@ module MeasurementsRecipesBaseExt
 using Measurements: Measurement, value, uncertainty
 using RecipesBase
 
-@recipe function f(y::AbstractArray{<:Measurement})
-    yerror := uncertainty.(y)
+unrecognised_uncertainty_plot_message = """
```

Comment:
If anything this should be a `const`
```suggestion
const unrecognised_uncertainty_plot_message = """
```